### PR TITLE
Add Task 3

### PR DESCRIPTION
**Pull request by IvanFedarenka**

### CODE REVIEW

File: src/test/java/MailTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        driver.findElement(START_LOGIN_BUTTON).click();
+        driver.findElement(LOGIN_INPUT).sendKeys(LOGIN);
+        driver.findElement(LOGIN_INPUT).submit();
+        Thread.sleep(1000);
+
+        driver.findElement(PASSWORD_INPUT).sendKeys(PASSWORD);
+        driver.findElement(PASSWORD_INPUT).submit();
+        Thread.sleep(5000);
+
+        assertTrue(driver.findElement(SETTINGS_BUTTON).isDisplayed());
```
Comment (KBugrimCS) [inline src/test/java/MailTest.java:33]: Please add more assertions to this test

File: src/test/java/MailTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @AfterEach
+    public void stop() {
+    driver.quit();
```
Comment (KBugrimCS) [inline src/test/java/MailTest.java:38]: tabulation symbol

File: src/main/java/com/coherent/mail/locators/Locators.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package com.coherent.mail.locators;
+
+import org.openqa.selenium.By;
+
+public class Locators {
+
+    public static final String BASE_URL = "https://mail.yandex.com";
+    public static final String LOGIN = "SeleniumTestUser1";
+    public static final String PASSWORD = "Selenium_22";
```
[the remark this answers is not in the document]
Comment (Harkness30) [reply src/main/java/com/coherent/mail/locators/Locators.java]: replaced same as previous

File: src/test/java/MailTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Harkness30) [reply src/test/java/MailTest.java]: added for this and all new assertions

File: src/test/java/MailTest.java (hunk 2 above)
Comment (Harkness30) [reply src/test/java/MailTest.java]: done

**Pull request merged**